mysql-test/main/partition_innodb.result

```diff
@@ -245,10 +245,9 @@ KEY parent_id (parent_id)
 ) ENGINE=InnoDB;
 ALTER TABLE t1 PARTITION BY HASH (id) PARTITIONS 1;
 ALTER TABLE t1 ADD CONSTRAINT test_ibfk_1 FOREIGN KEY (parent_id) REFERENCES t2 (id);
-ERROR HY000: Partitioned tables do not support FOREIGN KEY
 ALTER TABLE t1 PARTITION BY HASH (id) PARTITIONS 2;
 ALTER TABLE t1 ADD CONSTRAINT test_ibfk_1 FOREIGN KEY (parent_id) REFERENCES t2 (id);
-ERROR HY000: Partitioned tables do not support FOREIGN KEY
+ERROR HY000: Can't create table `test`.`t1` (errno: 150 "Foreign key constraint is incorrectly formed")
```

Comment:
The error message looks incorrect here. Shouldn’t this fail because a duplicate constraint name `test_ibfk_1`? Would it work with a different constraint name?